#
# Gyro message
#
Header header
#
# rotationRate is zero when there is no rotation. as
# the rate of rotation increases, rotationRate increases
# from 0
#
int16 rotationRate
#
# L or R
#
string rotationDirection
